MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1152 @@
+---
+layout: post
+title: Editors in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn here all about Editors support in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.
```

Comment:
Learn about the Editors support in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.

Reply:
title description changed